app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -229,4 +232,12 @@ def cloud_volume_types
       )
     end
   end
+
+  def availability_zones
+    # Single availability zone per PowerVS Cloud Manager
```

Comment:
This looks a little strange, but _there can be only one_ availability zone per PowerVS service instance. At first we didn't implement it, but now I'm noticing that existing Cloud Volume logic makes use of it (and maybe elsewhere?).

Reply:
@jaywcarman how about using the service guid aka `cloud_manager.uid_ems` for the az.ems_ref?  That way if someone changes the name of the provider we don't delete & re-create the AZ record.

Reply:
Done. Nice catch!